Translations/Language.ro.xml

```diff
@@ -1388,251 +1388,251 @@
+    <entry lang="ro" key="IDC_ENABLE_EMV_SUPPORT">Activare suport EMV</entry>
+    <entry lang="ro" key="COMMAND_APDU_INVALID">Comanda APDU trimisă spre card nu este validă.</entry>
+    <entry lang="ro" key="EXTENDED_APDU_UNSUPPORTED">Comenzile extinse APDU nu pot fi folosite cu token-ul actual.</entry>
+    <entry lang="ro" key="SCARD_MODULE_INIT_FAILED">Eroare la încărcarea librăriei WinSCard / PCSC.</entry>
+    <entry lang="ro" key="EMV_UNKNOWN_CARD_TYPE">Cardul din cititor nu este un card EMV suportat.</entry>
+    <entry lang="ro" key="EMV_SELECT_AID_FAILED">AID-ul cardului din cititor nu a putut fi selectat.</entry>
+    <entry lang="ro" key="EMV_ICC_CERT_NOTFOUND">Certificatul de cheie publică ICC nu a fost găsit în card.</entry>
+    <entry lang="ro" key="EMV_ISSUER_CERT_NOTFOUND">Certificatul cheii publice al emitentului nu a fost găsit în card.</entry>
+    <entry lang="ro" key="EMV_CPLC_NOTFOUND">CLPC nu a fost găsit în cardul EMV.</entry>
+    <entry lang="ro" key="EMV_PAN_NOTFOUND">Nu s-a găsit niciun număr de cont principal (PAN) în cardul EMV.</entry>
+    <entry lang="ro" key="INVALID_EMV_PATH">Calea EMV nu este validă.</entry>
+    <entry lang="ro" key="EMV_KEYFILE_DATA_NOTFOUND">Nu se poate crea un fișier-cheie din datele cardului EMV.\n\nUnul dintre următoarele lipsește:\n- Certificatul cheii publice ICC.\n- Certificatul cheii publice al emitentului.\n- Date CPCL.</entry>
+    <entry lang="ro" key="SCARD_W_REMOVED_CARD">Niciun card în cititor.\n\nAsigurați-vă că acesta este introdus corect.</entry>
+    <entry lang="ro" key="FORMAT_EXTERNAL_FAILED">Comanda Windows format.com nu a reușit să formateze volumul ca NTFS / exFAT / ReFS: Eroare 0x%.8X.\n\nRevenire la utilizarea API-ului Windows FormatEx.</entry>
+    <entry lang="ro" key="FORMATEX_API_FAILED">API-ul Windows FormatEx nu a reușit să formateze volumul ca NTFS / exFAT / ReFS.\n\nStarea eșecului = %s.</entry>
+    <entry lang="ro" key="EXPANDER_WRITING_RANDOM_DATA">Se scriu datele aleatorii în spațiul nou...\n</entry>
+    <entry lang="ro" key="EXPANDER_WRITING_ENCRYPTED_BACKUP">Se scrie copia de rezervă a antetului recriptat...\n</entry>
+    <entry lang="ro" key="EXPANDER_WRITING_ENCRYPTED_PRIMARY">Se scrie antetul primar recriptat...\n</entry>
+    <entry lang="ro" key="EXPANDER_WIPING_OLD_HEADER">Se distruge vechiul antet de rezervă...\n</entry>
+    <entry lang="ro" key="EXPANDER_MOUNTING_VOLUME">Se montează volumul...\n</entry>
+    <entry lang="ro" key="EXPANDER_UNMOUNTING_VOLUME">Se demontează volumul...\n</entry>
+    <entry lang="ro" key="EXPANDER_EXTENDING_FILESYSTEM">Se extinde sistemul de fișiere...\n</entry>
   </localization>
-  <xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:xs="http: /  / www.w3.org / 2001 / XMLSchema">
```

Comment:
It seems you have done some kind of find and replace on '/' to ' / ', I don't know if that is the grammar norm in Romanian, but it seems to have caught an URL here.

Reply:
I agree with @Jertzukka. There so many places where spaces were added before and after `/` character. The URL part is definitely wrong but other places also seem suspecious. @TigerxWood can you please clarify why so many spaces were added around `/` character occurences?